packages/esm-patient-labs-app/src/lab-orders/add-lab-order/useTestTypes.ts

```diff
@@ -33,8 +34,14 @@ export function useTestTypes(): UseTestType {
     }));
   }, [data]);
 
+  const filteredTestTypes = useMemo(() => {
```

Comment:
Longer discussion: Obviously we can do some of this searching on the backend (i.e., it's a fixable bug that https://dev3.openmrs.org/openmrs/ws/rest/v1/concept?class=Test&q=a doesn't do the right thing) and here, because we're searching for concepts it _may_ be better to do this on the backend (since occasionally we want to resolve names that are synonyms or index terms and hence not displayed). However, for many domains, because of how we handle i18n from the backend, we actually need to do purely frontend searching like this, specifically, where we want to be able to search against the `display` text for a result from the REST API.